### ISSUE TITLE
Necessary rules extenstions

### ISSUE THREAD

**Issue by greenlikeorange** [kaungmyatlwin/myanmar-phonenumber-js#7]
I found google's phone number library using phone number list from ITU and I found ITU have well documented Myanmar list describe in the link https://www.itu.int/oth/T0202000092/en

How about adding those rules here.

**Comment by greenlikeorange** [comment]
Are you going to add those rules or I'm going to create new repository :C

**Comment by kaungmyatlwin** [comment]
Yes. A rule based database would be better to add it as a dependency to this project. Thank you!
